packages/hooks/src/useCurrencyConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { useFetch } from '@deriv/api';
+
+const useCurrencyConfig = (currency: string) => {
+    const { data, ...rest } = useFetch('website_status');
+    const currency_config = data?.website_status?.currencies_config?.[currency];
+    const modified_currency_config = currency_config
+        ? {
+              ...currency_config,
+              is_crypto: currency_config.type === 'crypto',
+              is_fiat: currency_config.type === 'fiat',
+              is_aud: currency === 'AUD',
+              is_usd: currency === 'USD',
+              is_eur: currency === 'EUR',
+              is_gbp: currency === 'GBP',
+              is_eth: currency === 'ETH',
+              is_btc: currency === 'BTC',
+              is_ltc: currency === 'LTC',
+              is_usdt: currency === 'USDT',
+              is_usdc: currency === 'USDC',
+              is_eurs: currency === 'EURS',
+              is_eusdt: currency === 'eUSDT',
+              is_tusdt: currency === 'tUSDT',
+              is_usdk: currency === 'USDK',
+              is_idk: currency === 'IDK',
+              is_pax: currency === 'PAX',
+              is_tusd: currency === 'TUSD',
+              is_ust: currency === 'UST', //? should map to USDT instead?
```

Comment:
<img width="622" alt="image" src="https://github.com/binary-com/deriv-app/assets/72082844/fa660e5f-0f20-4965-b352-389051a933e7">

I see some logic in the legacy code where we consider `UST` as `USDT`, Should we map it here in the hook if it's something we should always do? 🤔

Reply:
As checked with @nijil-deriv, We should map `UST` to `USDT`.